modules/reindex/build.gradle

```diff
@@ -121,6 +121,10 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
            baseDir,
            unzip.temporaryDir,
            version == '090'
+      waitCondition = { fixture, ant ->
+        return fixture.portsFile.exists()
```

Comment:
why is this now needed?

Reply:
The OldElasticsearch fixture used in the reindex module tests starts an Elasticsearch instance and waits for its HTTP service to be started, then it writes the HTTP port to the `ports` file.

Before this change, the fixture here used the default AntFixture's waitCondition which is to execute a HTTP GET request on the address indicated in the fixture's `ports` file. Most fixtures write a IP address and a port number (ex: 127.0.0.1:48356) in this file but OldElasticsearch only writes the Elasticsearch HTTP port. Without this change, the fixture executes a `GET http://9200` request and fails directly, which was unnoticed because the bug in the `waitCondition(this, ant) == false` evaluation.

I'll revert this change in this PR and address all changes related to  `waitCondition(this, ant) == false`  in another PR as your suggested.